news/2024/2024-03-03-4dmwc-2024-concludes.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: 4 Digit osu!mania World Cup 2024 Concludes
+date: 2024-03-03 12:00:00 +0000
+---
+
+The 4 Digit osu!mania World Cup concludes yet another iteration! Find out what the next generation of osu!mania beasts had to offer, fighting tooth and nail for a chance to prove themselves.
+
+![](/wiki/Tournaments/4DM/2024/img/4dm2024-banner.jpg)
+
+For anyone vaguely involved in community affairs, the 4 Digit osu!mania World Cup (or 4DM for short) should ring a familiar bell. It is one of the biggest community-run tournaments in the game with **over 200 participating players**, and a stepping stone for any player to overcome looking to aim for higher heights in the tournament sphere. So who knows, maybe we'll get to see our participants in this year's [osu!mania 4K World Cup](/wiki/Tournaments/MWC)!
+
+But before we delve into the future, let's recap what's been going on these last couple months, shall we?
+
+## Summary
+
+![](/wiki/shared/news/2024-03-03-4dmwc-2024-concludes/4dmwc-24-podium.jpg)
+
+First and foremost, congratulations to Vietnam for winning this year's 4DM! A feat to be proud of given how competitive these tournaments get. Their performance across the entire tournament was noteworthy, and if the last match's result is anything to go by, their overall superiority over their peers is worth highlighting.
+
+The United States and Taiwan were forces to be reckoned with. While the win may've appeared to be uncontested, that couldn't be further from the truth. Both teams' performances were unparalleled, and their podium placements were well deserved!
+
+We suggest watching the VOD if you haven't yet to get a taste of what the best of the best of players had to offer in their last match.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2073463371&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Suffice to say, if the 6th iteration of the 4DM could be defined with one word alone, that'd be "experimentation" with over **67 custom maps** made for this tournament alone! There were a lot of noteworthy changes and unique quirks that have set this tournament apart from others. The biggest of which would be the new pooling paradigm proposed. We can only hope for our expectations to be subverted yet again next year!
```

Comment:
same here

Reply:
There was another comment made on this paragraph. Besides the pooling paradigm stuff, what else had to be changed? It's not highlighted.

Reply:
ah, the last sentence, same as https://github.com/ppy/osu-wiki/pull/11053#discussion_r1510117296

Reply:
Discussed with the hosts. It is uncertain. Therefore, we can leave it ambiguous like "see you next year" or something of the sorts. Will use that for the time being.